setup.py

```diff
@@ -21,6 +21,7 @@
 
 
 install_requires = [
+    "boto3>=1.1,<1.2",
```

Comment:
Sorry for not pointing this out earlier, I think we should avoid adding ``boto3`` as a hard dependency as I expect most people won't be using cloudfront.